### PR TITLE
Proposal: Invite @Contributors to StackStorm Github special group

### PR DESCRIPTION
**Pull request by armab**
The proposal is to invite existing StackStorm Contributors into the special group within StackStorm Github org, eg. mention them via @StackStorm/contributors as well as give them more possibilities and permissions to help with the project day-to-day activities like triaging issues and reviewing PRs.
See https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#how-to-start-contributing valued contributor's activities for more info.

From the Contributor's side it may be a honor to be recognized by highlighting the StackStorm org membership and being part of the st2 community officially.

This is just a proposal and not sure if that makes sense for everyone. So besides of review from @StackStorm/tsc it would be nice to hear feedback and review from the existing Contributors as well: @amanda11, @nzlosh, @userlocalhost, @jjm, @trstruth, @winem.

### CODE REVIEW

**Comment by nmaludy** [comment]
I like this idea and like getting more people engaged to help out!

**Comment by jjm** [comment]
Sounds like a good proposal to me too.

**Comment by amanda11** [comment]
Sounds good to me.

**Comment by winem** [comment]
I'd really appreciate this and see it as a great chance to contribute to the project in different ways even when one does not have too much time to pick up large PRs for example:

- keep an eye on open issues and PRs and keep them up-to-date for a better overview
- follow up on very old and still open (maybe already solved) issues / PRs?
- reduce the load for the maintainers and keep their backs free for big issues :) 

What about a dedicated slack channel for any questions on this topic (close an issue? how to respond to x? and so on...) I guess this would be great to get us up to speed and ensure that we're all moving in the same direction.

File: OWNERS.md [hunk 1]
```diff
@@ -44,8 +44,8 @@ Being part of Technical Steering Committee (TSC) [@StackStorm/maintainers](https
 
 # Contributors
 Contributors are using and occasionally contributing back to the project, might be active in conversations or express their opinion on the project’s direction.
-They're not part of the TSC voting process, but appreciated for their contribution, involvement and may become Maintainers in future.
-<!-- [@StackStorm/contributors](https://github.com/orgs/StackStorm/teams/contributors) are invited to StackStorm Github organization and we appreciate their contribution and involvement. -->
+They're not part of the TSC voting process, but appreciated for their contribution, involvement and may become Maintainers in the future depending on their effort and involvement. See [How to become a Maintainer?](https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#how-to-become-a-maintainer)
+[@StackStorm/contributors](https://github.com/orgs/StackStorm/teams/contributors) are invited to StackStorm Github organization and have permissions to help triaging the Issues and reviewing PRs.
```
Comment (amanda11) [inline OWNERS.md:48]: Minor comment on wording, should it be "have permissions to help triage issues and review PRs". To me, "to help triaging the Issues" doesn't read quite correctly.

Comment (armab) [reply OWNERS.md]: Updated in https://github.com/StackStorm/st2/pull/5019/commits/e828eb870cd5bf264b6148a3c3af229de207e684
Thanks for the fixed version! :+1:

**Comment by armab** [comment]
Alright, after one week voting period we got `15/20` from the [TSC Member's](https://github.com/StackStorm/st2/blob/87233faa43b37fadce44fb9e29b4ba4e2a9fc3b9/OWNERS.md) voting pool which is more then required `2/3` votes according to the https://github.com/StackStorm/st2/blob/master/GOVERNANCE.md#how-decisions-are-made.

Thanks everyone for feedback, merging!

**Comment by armab** [comment]
Next we will create the new Github @StackStorm/contributors group with the permissions to triage issues and invite our current Contributors to help Maintainers.
Let's see how that works.

**Comment by armab** [comment]
Current https://github.com/StackStorm/st2/blob/master/OWNERS.md#contributors @nzlosh, @userlocalhost, @jjm, @trstruth, @winem were now invited to StackStorm org and @StackStorm/contributors team with the `Triage` permissions.

**Comment by armab** [comment]
@winem @trstruth you need to accept the Github invitation to join StackStorm org that came to your email associated with Github account.

**Pull request merged**